packages/mds-provider-service/@types/index.ts

```diff
@@ -16,11 +16,14 @@ export interface ProviderDomainModel {
 }
 
 export type ProviderDomainCreateModel = DomainModelCreate<ProviderDomainModel>
+export interface GetProvidersOptions {
+  provider_types?: PROVIDER_TYPE[]
+}
 
 export interface ProviderService {
   createProviders: (mdsProviders: ProviderDomainCreateModel[]) => ProviderDomainModel[]
   createProvider: (mdsProvider: ProviderDomainCreateModel) => ProviderDomainModel
-  getProviders: () => ProviderDomainModel[]
+  getProviders: (options: GetProvidersOptions) => ProviderDomainModel[]
```

Comment:
When options are partial, I usually make the whole thing optional as well...
`getProviders: (options?: GetProvidersOptions) => ProviderDomainModel[])`
You then need to destructure from `options ?? {}` on the server side handler.

Reply:
Oh nifty, I didn't know you could have optional parameters.